org.eclipse.xtext.xtext.ui.examples/projects/domainmodel/org.eclipse.xtext.example.domainmodel.releng/BUILD.md

```diff
@@ -0,0 +1,46 @@
+# Build
+
+This project uses Maven with Eclipse Tycho. To perform a build open a command line and simple run 
```

Comment:
simple run -> simply run.

Overall, this can be simplified:

Open a command line an run `mvn` to build the project.